picamera2/controls.py

```diff
@@ -156,5 +156,5 @@ def defaults(self) -> dict:
 
     @property
     def reset(self) -> None:
-        for ctrl, val in self.defaults().items():
+        for ctrl, val in self.defaults.items():
```

Comment:
Could this be squashed into the original commit that introduced this code? Again, it just makes the review job a little bit easier. Thanks!

Reply:
Can do. Just need some input on other things before I force push some changes.